models/intermediate/int_iterable__list_user_unnest.sql

```diff
@@ -0,0 +1,66 @@
+with latest_user as (
+
+    select * 
+    from {{ ref('int_iterable__latest_user') }}
```

Comment:
realizing that this does not 100% recreate the list_user_history table since we draw from the latest user history record --> should we change this to select from the source user_history table instead?